docs/integrate/status.md

```
@@ -282,6 +282,8 @@ to CrateDB.
 <b>SDK</b>
 </td>
 <td>
+<a href="https://github.com/crate/commons-codec/actions/workflows/tests.yml" target="_blank">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/commons-codec/tests.yml?branch=main&label=commons-codec" loading="lazy"></a>
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)